.travis.yml

```diff
@@ -16,9 +16,16 @@ script:
 - docker run --env-file .env.test todo-app:test todo_app/tests/test_app.py
 - docker run --env TRELLO_BOARD_ID --env TRELLO_AUTH --env TRELLO_TOKEN todo-app:test todo_app/tests_e2e/test_e2e.py
 - echo $DOCKER_PASSWORD | docker login --username $DOCKER_USER --password-stdin
```

Comment:
No harm, but this line might as well move into the before_deploy stage since it isn't used unless we're deploying